test/refactoring/ConvertAsync/convert_protocol_requirement.swift

```diff
@@ -0,0 +1,11 @@
+protocol Foo {
+// RUN: swift-refactor --convert-to-async -source-filename %s -pos=%(line+1):6 | %FileCheck %s
+  func bar(for number: Int, completion: @escaping (_ success: Bool) -> Void)
+}
+// FIXME: The current async refactoring only refactors the client side and thus only adds the 'async' keyword.
```

Comment:
There's a test for this in basic.swift but it is only checking add-async-alternative, could just add another RUN line for convert-to-async there.

Reply:
I didn’t see that. Thanks. Moved the test case to `basic.swift`.